src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -36,7 +35,8 @@ private static SafeTerminateHandler<TraceEventSession> MakeSafeTerminateTraceEve
         /// <returns></returns>
         public static void Record(string assemblyFileName, string runId, string outputDirectory, Action action, Action<string> collectOutputFilesCallback)
         {
-            if (!IsRunningAsAdministrator)
+            bool? isElevated = TraceEventSession.IsElevated();
+            if (!isElevated.HasValue || isElevated.Value == false)
```

Comment:
This is a call to the Tracing library API and the code looks like this:
```csharp
// TODO FIX NOW make these private 
internal static bool? IsElevated()
{
#if !NOT_WINDOWS 
    Process process = Process.GetCurrentProcess();
    IntPtr tokenHandle = IntPtr.Zero;
#if DOTNET_CORE // TODO FIX use SafeHandles everywhere, but we have to decide how far back we want to support. 
    if (!OpenProcessToken(process.SafeHandle.DangerousGetHandle(), TOKEN_QUERY, out tokenHandle))
        return null;
#else
    if (!OpenProcessToken(process.Handle, TOKEN_QUERY, out tokenHandle))
        return null;
#endif
    int tokenIsElevated = 0;
    int retSize;
    bool success = GetTokenInformation(tokenHandle, TOKEN_INFORMATION_CLASS.TokenElevation, (IntPtr)(&tokenIsElevated), 4, out retSize);
    CloseHandle(tokenHandle);
    if (!success)
        return null;
    GC.KeepAlive(process);      // TODO get on SafeHandles. 
    return tokenIsElevated != 0;
#else
    return true;
#endif // !NOT_WINDOWS
}
```
If the API fails to get information about elevation, then it returns NULL, otherwise it returns TRUE|FALSE. We have not tested on lower than Window 10, but in theory the API should work on Windows 8, and I do not know what happens in lower versions of Windows. Is this something we should support?